src/wordpress-recommendations.lib.php

```diff
@@ -26,70 +25,244 @@
+         */
+        if (!is_ssl()) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['noSSL'] = array(
+                __('Implement a SSL Certificate', 'sucuri-scanner') => __('SSL certificates help protect the integrity of the data in transit between the host (web server or firewall) and the client (web browser).', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if PHP version being used needs to be upgraded.
+         * @see https://www.php.net/supported-versions.php
+         */
+        if (version_compare(phpversion(), '7.2', '<')) {
+            // phpcs:disable Generic.Files.LineLength
             $recommendations['PHPVersionCheck'] = array(
-                __('Upgrade PHP to a supported version', 'sucuri-scanner') =>
-                __('The PHP version you are using no longer receives security support and could be exposed to unpatched security vulnerabilities.', 'sucuri-scanner')
+                __('Upgrade PHP to a supported version', 'sucuri-scanner') => __('The PHP version you are using no longer receives security support and could be exposed to unpatched security vulnerabilities.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if WordPress Salt & Security Keys are set and were updated on the last 6 months.
+         * @see https://wordpress.org/support/article/editing-wp-config-php/#security-keys
+         * @see https://sucuri.net/guides/wordpress-security/#harrec
+         */
+        if (!defined('AUTH_KEY') || !defined('AUTH_SALT')) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['wpSaltExistenceChecker'] = array(
+                __('Missing WordPress Salt & Security Keys', 'sucuri-scanner') => __('Consider using WordPress Salt & Security Keys to add an extra layer of protection to the session cookies and credentials.', 'sucuri-scanner'),
+            );
+        // phpcs:enable
+        } elseif (file_exists(ABSPATH.'/wp-config.php')) {
+            if (filemtime(ABSPATH.'/wp-config.php') < strtotime('-6 months')) {
+                // phpcs:disable Generic.Files.LineLength
+                $recommendations['wpSaltAgeDiscriminator'] = array(
+                    __('WordPress Salt & Security Keys should be updated', 'sucuri-scanner') => __('Updating WordPress Salt & Security Keys after a compromise and on a regular basis, at least once a year, reduces the risks of session hijacking.', 'sucuri-scanner'),
+                );
+                // phpcs:enable
+            }
+        }
+
+        /*
+         * Check if there is a standard administrator/admin account.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
+         */
+        if (!empty(get_users(array('role' => 'administrator', 'search' => 'admin*')))) {
```

Comment:
Better to check specifically for login name admin and administrator. The `'search' => 'admin*'` will give us results like 'admin_joe185679' and will also search other fields like email addresses.